.github/workflows/ci.yml

```diff
@@ -173,7 +173,8 @@ jobs:
           set -x
           mkdir build && cd build
           cmake -DCMAKE_CXX_STANDARD=${{ matrix.std }} -DCMAKE_BUILD_TYPE=${{ matrix.build_type }} \
-            -DCMAKE_CXX_COMPILER=${{ matrix.cxx_compiler }} -DCMAKE_C_COMPILER=${{ matrix.c_compiler }} -DTBB_CPF=${{ matrix.preview }} ..
+            -DCMAKE_CXX_COMPILER=${{ matrix.cxx_compiler }} -DCMAKE_C_COMPILER=${{ matrix.c_compiler }} -DTBB_CPF=${{ matrix.preview }} \
+            -DCMAKE_VISIBILITY_INLINES_HIDDEN=TRUE -DCMAKE_C_VISIBILITY_PRESET=hidden -DCMAKE_CXX_VISIBILITY_PRESET=hidden ..
```

Comment:
I don't think that this is a good idea to add these visibility flags everywhere. Lets make only one-two instances of test matrix to actually set these flags.

Reply:
I think we need this flags on every platform for one of ``test`` and ``example`` build.
I'll change this commit after reviewing other changes.

Reply:
But, I think that the use of these flags for all configurations will make it easier to identify export of symbols errors in the project.

Reply:
agree with @phprus 
it's a good practice to always set visibility to hidden and explicitly export required symbols. 

But it's better to set such flags in cmake scripts, not in CI.

Reply:
First export issue found by this cmake options:
https://github.com/oneapi-src/oneTBB/actions/runs/5164552958/jobs/9303530791?pr=1114

Thanks @ilya-lavrenov